index.html

```diff
@@ -201,10 +201,10 @@ <h2>About me</h2>
 
     <section id="ContactForm">
       <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-      <form>
-        <input class="styleForm" type="text" id="fname" placeholder="Full name">
-        <input class="styleForm" type="email" id="email" placeholder="Email address">
-        <textarea type="text" id="message" placeholder="Write me something..."></textarea>
+      <form action="https://formspree.io/f/mwkjdrqv" method="POST">
+        <input class="styleForm" name="name" type="text" id="fname" placeholder="Full name" required maxlength="30">
+        <input class="styleForm" name="email" type="email" id="email" placeholder="Email address" required>
+        <textarea type="text" id="message" name="message" placeholder="Write me something..." required maxlength="500"></textarea>
         <button class="projectButton">Get in touch</button>
```

Comment:
According to the project requirements, it is important that we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Kindly implement the button interactions such as hover, active (when the button is pressed), disabled, and enable (default state) styled as shown in the Figma template. 🤗🛠

![image](https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/78251927/8e9ffad4-0825-4a21-9343-f7b58535aa00)